### PR TITLE
masks from (n,h,w) to [(h,w),...,(h,w)]

### PR DESCRIPTION
**Pull request by howardchina**
https://github.com/albumentations-team/albumentations/blob/master/albumentations/core/composition.py#L223
"data" is List

### CODE REVIEW

File: mmdet/datasets/pipelines/transforms.py [hunk 1]
```diff
@@ -1215,7 +1215,11 @@ def __call__(self, results):
                 raise NotImplementedError(
                     'Albu only supports BitMap masks now')
             ori_masks = results['masks']
-            results['masks'] = results['masks'].masks
+            masks_list = []
+            bitmapmasks = results['masks'].masks
+            for i in range(bitmapmasks.shape[0]):
+                masks_list += [bitmapmasks[i,:,:]]
+            results['masks'] = masks_list
```
Comment (yhcao6) [inline mmdet/datasets/pipelines/transforms.py:1222]: Can be simplified by [`np.vsplit`](https://numpy.org/doc/stable/reference/generated/numpy.vsplit.html)

**Comment by hellock** [comment]
Task linked: [CU-4zxm6k [Review] MMdet #4032](https://app.clickup.com/t/4zxm6k)

**Comment by yhcao6** [comment]
@howardchina, please sign CLAassistant, then we can merge.

**Comment by howardchina** [comment]
CLAassistant is signed already.

**Review by ZwwWayne — APPROVED** [review]

**Comment by Adblu** [comment]
Thanks for the fantastic work regarding new release !
I have 1 question regarding TODO in [transforms.py ](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/datasets/pipelines/transforms.py#L1333), my training crushes when I'm using albu augmentation. I have found a bug [fix](https://github.com/open-mmlab/mmdetection/issues/4220), however its not elegant and brings lot of problems. 
Are you planning to resolve it in future @howardchina  @ZwwWayne @yhcao6 ?

**Comment by ZwwWayne** [comment]
Please @yhcao6 have a look for that.

**Comment by Adblu** [comment]
@ZwwWayne @yhcao6 Is there any update on the issue ?

**Pull request merged**